packages/blockchain-wallet-v4-frontend/src/modals/Swap/EnterAmount/template.failure.tsx

```diff
@@ -18,6 +20,11 @@ const Wrapper = styled.div`
 `
 
 const Failure: React.FC<Props> = (props) => {
+  const errorCallback = () => props.swapActions.setStep({ step: 'INIT_SWAP' })
```

Comment:
I suggest to use `returnToInitSwap` here instead. In case if not quote price failed, on navigation back we want to stop and reset it.